tests/test_markdown.py

```diff
@@ -306,5 +306,49 @@ def test_remove_license(self, base_filename, want_filename):
                 self.assertEqual(test_file.read(), mdfile_want.read())
 
 
+    test_markdown_filenames = [
+        [
+            "tests/markdown_example_base.md",
+            "tests/markdown_example_additional_content.md",
+            "tests/markdown_example_base_expected.md",
+        ],
+        [
+            # The content should be the same as base, and not throw any errors.
+            "tests/markdown_example_base.md",
+            "tests/markdown_example_does_not_exist.md",
+            "tests/markdown_example_dne_expected.md",
```

Comment:
Nit: I think it's a bit clearer if you leave the expected result as `tests/markdown_example_base.md` because it makes it's clear it's the same file (so it has the same contents).

Reply:
Done!